src/app/styles/components/addresses.scss

```diff
@@ -17,4 +17,18 @@
       margin-right: 0.3em;
     }
   }
+
+  &__link-overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
We can make 4 lines(from 23 to 26) into 1 line?

Reply:
I do not know how